src/if_python3.c

```diff
@@ -777,7 +792,7 @@ static int python_end_called = FALSE;
     static void
 call_PyObject_Free(void *p)
 {
-#if defined(Py_DEBUG) && !defined(Py_DEBUG_NO_PYMALLOC)
+# if (defined(Py_DEBUG) || PY_VERSION_HEX >= 0x030800f0) && !defined(Py_DEBUG_NO_PYMALLOC)
```

Comment:
```suggestion
#if (defined(Py_DEBUG) || PY_VERSION_HEX >= 0x030800f0) && !defined(Py_DEBUG_NO_PYMALLOC)
```

Reply:
Sorry, overlooked that - committing right away.